targets/ESP32/_nanoCLR/nanoFramework.Espnow/nanoFramework_espnow_native.h

```
@@ -0,0 +1,140 @@
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#ifndef _NANOFRAMEWORK_EspNow_NATIVE_H_
+#define _NANOFRAMEWORK_EspNow_NATIVE_H_
```

Clang (14.0.6)

[warning] declaration uses identifier '_NANOFRAMEWORK_EspNow_NATIVE_H_', which is a reserved identifier

(bugprone-reserved-identifier)